layouts/page/resources.html

```diff
@@ -20,16 +21,27 @@ <h2 class="text-center text-primary fw-lighter">
                 </li>
                 {{ $first = false }}
             {{ end }}
+
+            <li class="nav-item ms-auto">
+                <a class="nav-link text-decoration-underline" href="https://drive.google.com/drive/folders/1rjg6h0bArc4QbKTAdAV6O8mYe5dVYlil?usp=drive_link" target="_blank">Drive Link</a>
```

Comment:
May you put this link into `config.toml`?

Reply:
hey jared, thanks for the suggestion! Lemme try it out on my end